src/content/docs/contributing/client-api/auth.md

```diff
@@ -2,6 +2,27 @@
 title: "Package: auth"
 docType: client-package
 package:
+  interfaces:
+    - name: AuthRequest
+      description: |
+        Object used to perform authenticated HTTP requests. 
+        In languages where such constructs are already provided (e.g. python, see [1]), they should be used instead of this object to take
+        advantage of the existing ecosystem.
+        
+        This can be created from the [OAuth2ClientCredentials](#class-OAuth2ClientCredentials) Object by using [oauth2AuthRequest](#functions-oAuth2AuthRequest).
```

Comment:
```suggestion
        This can be created from the [OAuth2ClientCredentials](#class-OAuth2ClientCredentials) Object by using [oauth2AuthRequest](#function-oAuth2AuthRequest).
```
my bad I gave you faulty code 😭